server/server.js

```diff
@@ -100,4 +98,19 @@ app.get('/tasks', (req, res) =>
         })
 );
 
+app.post('/taskUpdated', (req, res) => {
```

Comment:
It's not a good practice to use camelCase in the endpoint names, use e.g `/task/update` instead